test/e2e/storage/volume_provisioning.go

```diff
@@ -903,6 +1008,55 @@ func newBetaStorageClass(t storageClassTest, suffix string) *storagebeta.Storage
 	}
 }
 
+func startGlusterDpServerPod(c clientset.Interface, ns string) *v1.Pod {
+	podClient := c.CoreV1().Pods(ns)
+
+	provisionerPod := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "glusterdynamic-provisioner-",
+		},
+
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:  "glusterdynamic-provisioner",
+					Image: "quay.io/humblec/glusterdynamic-provisioner:v1.0",
```

Comment:
make this in gcr.io

Reply:
Sure, is there a common repo or I can host it under my account ?

Reply:
Look at `test/images/*`, Kubernetes maintains Dockerfiles for images used in testing there. Someone from Google then needs to build and push the image to gcr.io.